cloud.foundry.cli/src/main/java/cloud/foundry/cli/services/DiffController.java

```diff
@@ -35,6 +38,40 @@ public Integer call() {
         return 0;
     }
 
+    @Command(name = "services", description = "Print the differences between " +
+            "the services given in the yaml file and the configuration of the services of your cf instance.")
+    static class DiffServiceCommand implements Callable<Integer> {
+
+        @Mixin
+        LoginCommandOptions loginOptions;
+
+        @Mixin
+        YamlCommandOptions yamlCommandOptions;
+
+        @Override
+        public Integer call() throws Exception {
+            DefaultCloudFoundryOperations cfOperations = CfOperationsCreator.createCfOperations(loginOptions);
+            ServicesOperations servicesOperations = new ServicesOperations(cfOperations);
+
+            SpecBean specBeanDesired = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
```

Comment:
We have talked about this and even though we came to the conclusion that it's not important, since it's only a temporary implementation, it would probably still be a good choice to set space developers and apps to null, so that they don't get compared, just to be consistent with what the operation description suggests.
```suggestion
            SpecBean specBeanDesired = YamlMapper.loadBean(yamlCommandOptions.getYamlFilePath(), SpecBean.class);
            specBeanDesired.setSpaceDevelopers(null);
            specBeanDesired.setApps(null);
```

Also would be awesome if the same goes for the diff application command. Since i wrote that code, i could change that also. :)